luigi/__meta__.py

```diff
@@ -7,5 +7,5 @@
 __author__ = 'The Luigi Authors'
 __contact__ = 'https://github.com/spotify/luigi'
 __license__ = 'Apache License 2.0'
-__version__ = '2.8.13'
+__version__ = '3.0.0b2'
```

Comment:
@honnix shouldn't we change version here prior to merge?

Reply:
Good catch. We can release 3.0.0 immediately after merging to master. So it is OK in this PR I think.